examples/checkout/sdk-switch-network-with-nextjs/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "sdk-connect-with-nextjs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "test": "playwright test"
+  },
+  "dependencies": {
+    "@biom3/react": "^0.25.21",
+    "@ethersproject/providers": "^5.7.2",
+    "@imtbl/sdk": "^1.48.1",
```

Comment:
IMO it would be best to switch the version to "latest" so we can always have this tested and running against the latest sdk version.